docs/installation/debian.md

```diff
@@ -49,17 +49,21 @@ from the new repository:
          $ apt-get purge lxc-docker*
          $ apt-get purge docker.io*
 
- 4. Add the new `gpg` key.
+ 4. Ensure that APT can work with `https` method and that CA certificates are installed.
+
+         $ apt-get install apt-transport-https ca-certificates
```

Comment:
well, maybe we should, because we don't update these packages later on, and we don't want outdated packages.